actions/promote/postCopyWorker.js

```diff
@@ -123,12 +123,10 @@ async function previewPublish(doPublish, batchManager) {
         // Write the information to batch manifest
         currentBatch.writeResults({ failedPromotes, failedPreviews, failedPublishes });
         throw new Error(stepMsg);
-    } else {
-        stepMsg = `Promoted floodgate for ${currBatchLbl} successfully`;
-        logger.info(stepMsg);
     }
     logMemUsage();
-    stepMsg = `All tasks for floodgate promote of ${currBatchLbl} is completed`;
+    logger.info(`All tasks for promote ${currBatchLbl} is completed`);
+    stepMsg = 'All tasks for floodgate promote is completed';
```

Comment:
@raga-adbe-gh Is this msg returned as part of the promote status API response? As this is a batch level completion message, a status message saying `All tasks for floodgate promote is completed` at batch level might be confusing. May be adding batch information when a batch is fully done with copy, preview and publish will be beneficial. Including total batch count will be even better.

eg:  `All tasks for floodgate promote is completed for batch ${currBatchLbl} / ${totalBatchCount}`

Reply:
Thanks @sukamat . The status on copyWorker  and postPromoteWorker are batch specific. But we also are not reporting the batch progress in overall status. Updated to show as suggested e.g. "Promoting batch 3 / 5."